Cuestionarios/Cuestionarios/Controllers/QuestionController.cs

```diff
@@ -13,6 +13,7 @@ public class QuestionController
     {
         readonly UnitOfWork iUOfW = new UnitOfWork();
 
+        //Save the questions
```

Comment:
usa sumary para los comentarios

Ejemplo
/// <summary>
/// Save the questions
/// </summary>